llvm/test/Feature/Repo/repo_hash_contributions_var.ll

```diff
@@ -0,0 +1,55 @@
+; This test is used to check the hash calculation for the contributions.
+;
+; All GOs' information are shown below:
+; GO Name: Var1, Initial Dependencies: [ Func1 ], Contributions: [ ]
+; GO Name: Var2, Initial Dependencies: [ ], Contributions: [ ]
+; GO Name: Func1, Initial Dependencies: [ ], Contributions: [ ]
+; GO Name: Main, Initial Dependencies: [ Func2 ], Contributions: [ Var2 ]
+; GO Name: Func2, Initial Dependencies: [ ], Contributions: [ Var1 ]
+;
+; This test checks that `Main` digest is changed when `Var2` is modified.
+;
+; The test invokes five steps:
+;  1) Compile the initial code `Inputs/repo_contributions_dependencies_loop.ll` to %t;
+;  2) Compile `repo_hash_contributions_var.ll` to %t1.
+;  3) Dump the digest of `Main` in %t.
+;  4) Dump the digest of `Main` in %t1.
+;  5) Check the digests of `Main` are different since the variable `Var2` is changed.
+;
+; RUN: rm -rf %t.db
+; RUN: env REPOFILE=%t.db clang -c -O0 --target=x86_64-pc-linux-gnu-repo -x ir %S/Inputs/repo_contributions_dependencies_loop.ll -o %t
+; RUN: env REPOFILE=%t.db clang -c -O0 --target=x86_64-pc-linux-gnu-repo -x ir %s -o %t1
+; RUN: env REPOFILE=%t.db repo-fragments %t Main -repo=%t.db -digest-only > %t.d
+; RUN: env REPOFILE=%t.db repo-fragments %t1 Main -repo=%t.db -digest-only > %t1.d
+; RUN: not diff %t.d %t1.d
+
+target triple = "x86_64-pc-linux-gnu-elf"
```

Comment:
repo rather than ELF? Do we need to specify both here _and_ on the command-lines?

Reply:
> 
> 
> repo rather than ELF? Do we need to specify both here _and_ on the command-lines?
>
Yes, should be repo. I tried to remove the `triple` in the command line and only keep `target triple = "x86_64-pc-linux-gnu-elf"` in the IR code, it didn't generated repo ticket after running the command.

I think if the clang's input file is the IR code, target triple need to be given in the command line. 

Removed  `target triple = "x86_64-pc-linux-gnu-elf"` in commit e3d5b7ca6.